src/components/header.js

```diff
@@ -297,6 +297,15 @@ const AboutContent = ({ visible }) => {
               </span>
             </li>
           </ul>
+          <strong className='d-block mb-3'>Authors and Contributors</strong>
+              <span className='d-block'>
+                <p>
+                  <a href='https://github.com/johnlees' >Dr John Lees</a>, 
+                  <a href='https://github.com/Danderson123' >&nbsp;Daniel Anderson</a>, and 
+                  <a href='https://github.com/bruhad-dave' >&nbsp;Bruhad Dave 
+                  </a>
```

Comment:
I think you want to exclude the spaces from the hyperlinks:
```suggestion
                  <a href='https://github.com/johnlees' >Dr John Lees</a>, 
                  &nbsp;<a href='https://github.com/Danderson123' >Daniel Anderson</a>, and 
                  &nbsp;<a href='https://github.com/bruhad-dave' >Bruhad Dave</a>
```